backend/models/cans.py

```diff
@@ -690,6 +693,26 @@ class CAN(BaseModel):
         "Project", secondary="project_cans", back_populates="cans"
     )
 
+    @property
+    def status(self):
+        if object_session(self) is None:
+            return CANStatus.INACTIVE
+        current_year = date.today().year
+        can_fiscal_year = object_session(self).scalar(
+            select(CANFiscalYear).where(
+                and_(
+                    CANFiscalYear.can_id
+                    == self.id,
+                    CANFiscalYear.fiscal_year == current_year
+                )
+            )
+        )
+        if can_fiscal_year is None:
+            return CANStatus.INACTIVE
+        can_status = CANStatus.INACTIVE if can_fiscal_year.expected_funding <= 0 and self.expiration_date < datetime.now() else CANStatus.ACTIVE
```

Comment:
I would create a method `can_fiscal_year.available_funding()` (or something like that) that uses the calculations I pointed you to this morning - it should be something like `total funding - amount spent`